app/lib/screens/HomeScreens/ChatList.dart

```diff
@@ -0,0 +1,192 @@
+// ignore_for_file: prefer_const_constructors, avoid_unnecessary_containers, sized_box_for_whitespace, prefer_final_fields
+
+import 'package:effektio/common/store/Colors.dart';
+import 'package:effektio/common/widget/ChatOverview.dart';
+import 'package:effektio/repository/client.dart';
+import 'package:effektio_flutter_sdk/effektio_flutter_sdk_ffi.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class ChatList extends StatefulWidget {
+  const ChatList({Key? key, required this.client}) : super(key: key);
+  final Future<Client> client;
+
+  @override
+  State<ChatList> createState() => _ChatListState();
+}
+
+class _ChatListState extends State<ChatList> {
+  String? userId = '';
+
+  @override
+  void initState() {
+    _getUser();
+    super.initState();
+  }
+
+  static const TextStyle optionStyle =
+      TextStyle(fontSize: 30, fontWeight: FontWeight.bold, color: Colors.white);
+
+  Future<void> _getUser() async {
+    final String? id = await getUser(widget.client);
+    setState(() {
+      userId = id;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Color.fromRGBO(36, 38, 50, 1),
+      appBar: AppBar(
+        leadingWidth: MediaQuery.of(context).size.width / 5,
+        toolbarHeight: MediaQuery.of(context).size.height / 15,
+        backgroundColor: Color.fromRGBO(36, 38, 50, 1),
+        elevation: 0,
+        leading: Container(
+          margin: const EdgeInsets.only(left: 10),
+          child: TextButton(
+            onPressed: () {},
+            child: Text(
+              'Select',
+              style: TextStyle(
+                fontSize: 14,
+                color: AppColors.primaryColor,
+              ),
+            ),
+          ),
+        ),
+        actions: [
+          IconButton(
+            onPressed: () {},
+            padding: const EdgeInsets.only(
+              right: 10,
+            ),
+            icon: SvgPicture.asset(
+              'assets/images/edit.svg',
+              width: 20,
+              height: 20,
+            ),
+          ),
+        ],
+      ),
+      body: Container(
+        child: SingleChildScrollView(
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              Container(
+                margin: const EdgeInsets.only(
+                  left: 18,
+                ),
+                child: Text(
+                  'Chat',
+                  style: TextStyle(
+                    color: Colors.white,
```

Comment:
Someone should explain to me, how theming is supposed to work with `MaterialTheme` in flutter because again and again I see people _hard define colors and sizes_ in the widgets rather than on the Theme. How are we suppose to replace the theme later (for branded versions of the App) if they are hard coded? Can we please have a guide on how to use the Theming and enforce that?!?

Reply:
There are various ways of doing this, we can setup ThemeData for this that will control all the theme of the app, so we don't need to hard code values in every widget. You can take the example of textTheme which I have implemented. This can go in a seperate PR.